### PR TITLE
DNM - just checking diff chore(lottery-v1): Remove all lottery v1 code from codebase

### PR DESCRIPTION
**Pull request by ChefHutch**
- Remove all v1 code
- Rename `LotteryV2` view components to `Lottery` (note - contracts still distinguished as v2)

### CODE REVIEW

File: src/views/Lottery/components/Hero.tsx [hunk 1]
```diff
@@ -1,104 +1,279 @@
 import React from 'react'
-import styled from 'styled-components'
-import { Heading, Text, LinkExternal } from '@pancakeswap/uikit'
+import styled, { keyframes } from 'styled-components'
```
Comment (ChefHutch) [inline src/views/Lottery/components/Hero.tsx:2]: This is a straight delete of v1 and replace with v2, but the file name is the same, so Git is parsing it as a diff

`views/Lottery/components/Hero` > deleted
`views/LotteryV2/components/Hero` >  renamed `views/Lottery/components/Hero`

**Review by Chef-Cheems — CHANGES_REQUESTED** [review]
Messaged in slack about additional removals

**Review by Chef-Cheems — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -55,7 +55,6 @@
     "lodash": "^4.17.21",
     "node-fetch": "^2.6.1",
     "react": "^17.0.1",
-    "react-chartjs-2": "2.11.1",
```
Comment (RabbitDoge) [inline package.json:58]: and `chart.js` ?

Comment (ChefHutch) [reply package.json:58]: Good point, removed

**Review by RabbitDoge — APPROVED** [review]

**Pull request merged**